Xamarin.Forms.Platform.iOS/FormsApplicationDelegate.cs

```diff
@@ -156,20 +168,18 @@ void CheckForAppLink(NSUserActivity userActivity)
 		void SetMainPage()
 		{
 			UpdateMainPage();
-			_window.MakeKeyAndVisible();
+			Window.MakeKeyAndVisible();
 		}
 
 		void UpdateMainPage()
 		{
 			if (_application.MainPage == null)
 				return;
 
-			var platformRenderer = (PlatformRenderer)_window.RootViewController;
-
-			if (platformRenderer != null)
+			if(Window.RootViewController is PlatformRenderer platformRenderer)
```

Comment:
This is C#7, yeah? Love it, but we're stuck in VS2015 land for now. :( 

Also, if you make other changes to the file, please also add a space between the if and parens. Not a big deal.

Reply:
Hmmm ok, i'll make some changes. There is nothing about this in the Wiki on how to contribute.